Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -408,6 +408,27 @@ extension Reactive where Base: FileManager {
 		}
 		.map { URL(fileURLWithPath: $0, isDirectory: true) }
 	}
+  
+  public func copyItem(_ source: URL, into: URL) -> SignalProducer<URL, CarthageError> {
+    let destination = into.appendingPathComponent(source.lastPathComponent)
+    do {
+      try self.base.copyItem(at: source, to: destination, avoiding·rdar·32984063: true)
+      return SignalProducer(value: destination)
+    } catch {
+      return SignalProducer(error: .internalError(description: "copyItem failed: \(error)"))
+    }
+  }
+  
+  public func replaceItem(at originalItemURL: URL, withItemAt newItemURL: URL) -> SignalProducer<(), CarthageError> {
+    do {
+      guard (try FileManager().replaceItemAt(originalItemURL, withItemAt: newItemURL, backupItemName: nil, options: .usingNewMetadataOnly)) != nil else {
```

Comment:
I missed this before, but this should be using `self.base`, not `FileManager()`

Reply:
Ah. You're right. Stand by…

Reply:
Done!